checks/cs50/2017/x/speller/checks.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Speller(Checks):
+
+    @check()
+    def exists(self):
+        """dictionary.c, dictionary.h, and Makefile exist"""
+        super(Speller, self).exists("dictionary.c")
+        super(Speller, self).exists("dictionary.h")
+        super(Speller, self).exists("Makefile")
```

Comment:
To avoid having contributors have to remember special syntax just for this one check, should we just rename the parent's method to `isfile`, a la https://docs.python.org/2/library/os.path.html#os.path.isfile? Then every check can just use `self.*`.